equinix/common_test.go

```diff
@@ -0,0 +1,120 @@
+package equinix
+
+import (
+	"fmt"
+	"strings"
+	"time"
+)
+
+// list of plans and metros and os used as filter criteria to find available hardware to run tests
+var (
+	preferable_plans  = []string{"x1.small.x86", "t1.small.x86", "c2.medium.x86", "c3.small.x86", "c3.medium.x86", "m3.small.x86"}
```

Comment:
Despite the diff, this file is not new content.  Everything in this file was copy-pasted from the device resource test file in order to avoid moving all of the remaining metal resources: https://github.com/equinix/terraform-provider-equinix/blob/611e78a32a865aa374d2e4896cd0978252b983c4/equinix/resource_metal_device_acc_test.go#L26